src/common/utils/pagination.ts

```
@@ -0,0 +1,37 @@
+
+  static calcTotalPage(total: number, limit: number): number {
+    return Math.ceil(total / limit);
+  }
+
+  static validatePage(page: number, limit: number, total: number): number {
+    const totalPages = this.calcTotalPage(total, limit);
+    if (page < 1) return 1;
+    if (page > totalPages) return totalPages;
+    return page;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)